README.md

```diff
@@ -14,15 +14,50 @@ Inspired by the [invidio.us](https://github.com/omarroth/invidious) project.
 ## Installation
 
 ```bash
-git clone https://github.com/zedeus/nitter
-cd nitter
-nimble build
+# useradd -m nitter
+# su nitter
+$ curl https://nim-lang.org/choosenim/init.sh -sSf | sh
```

Comment:
Instead of this, add a note above saying it requires a Nim installation with a link to the choosenim repo.

Reply:
I chose https://nim-lang.org/install.html instead